src/perl5/Bio/JBrowse/Cmd/FormatSequences.pm

```diff
@@ -590,6 +590,9 @@ sub openChunkFile {
             "$refInfo->{name}-$chunkNum.txt"
           );
 
+
+    $file =~ s![/\\\n]!_!g; # convert filesystem reserved characters to underscore
```

Comment:
why would there be a newline in the filename?

Reply:
filename is made from the ref name, which might have a newline in it possibly